Code/Tools/LuaIDE/Source/StandaloneToolsApplication.h

```diff
@@ -39,7 +44,7 @@ namespace StandaloneTools
     private:
         AZStd::string GetStoragePath() const;
         
-        bool LaunchDiscoveryService();
+        bool StartDebugService();
```

Comment:
Is there a stop debug service equivalent? Or is it more that everything just gets torn down and so does not need explicit stop/cleanup?

Reply:
Yep, the only thing this does is start listening on TCP for the Lua IDE and AzNetworking cleans that up on destruction.